api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+from api.citations.utils import display_absolute_url
+
+from tests.base import ApiTestCase
+from tests.factories import (
+    AuthUserFactory,
+    PreprintFactory,
+)
+
+
+class PreprintCitationsMixin(object):
+
+	def setUp(self):
+		super(PreprintCitationsMixin, self).setUp()
+		self.admin_contributor = AuthUserFactory()
+		self.published_preprint = PreprintFactory(creator=self.admin_contributor)
+
+	def test_unauthenticated_can_view_published_preprint_citations(self):
```

Comment:
test_unauthenticated_cannot_view_unpublished_preprint_citations would probably be good to add.